manifests/g/Getscreenme/Dashboard/1.0.16/Getscreenme.Dashboard.locale.en-US.yaml

```
@@ -0,0 +1,26 @@
+# Created using wingetcreate 1.1.2.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.2.0.schema.json
+
+PackageIdentifier: Getscreenme.Dashboard
+PackageVersion: 1.0.16
+PackageLocale: en-US
+Publisher: Getscreen.me
```

```suggestion
Publisher: Getscreen
```
Must match Control Panel